### PR TITLE
Updated content-card-5-black.html

### PR DESCRIPTION
**Pull request by Tapo41**

### CODE REVIEW

**Comment by Tapo41** [comment]
Screenshot_20230211_003715

File: Content Cards/src/content-card-5-black.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+    
+    <div
+      class="relative bg-white w-full h-[456px] overflow-hidden text-center text-[14px] text-silver font-dm-sans"
+    >
+      <div
+        class="absolute top-[24px] left-[27px] rounded-base bg-gray w-[261px] h-[376px] bg-black rounded-t-3xl rounded-b-3xl"
+      ></div>
+      <img
+        class="absolute top-[24px] left-[28px] rounded-b-2xl rounded-t-2xl w-[261px] h-[168px] object-cover"
```
Comment (Rajdip019) [inline Content Cards/src/content-card-5-black.html:9]: Don't use absolute. It's in normal HTML flow. We don't need to use positioning here. @Tapo41

**Comment by Tapo41** [comment]
@Rajdip019 Updated .
Check it out.

**Comment by Tapo41** [comment]
@RavindraP04  Done
Screenshot_20230211_221040

**Pull request merged**